### PR TITLE
Eri9600/spatial relationships

### PR DESCRIPTION
**Pull request by eri9600**
Spatial Relationships Sample as shown here: https://devtopia.esri.com/runtime/common-samples/issues/197

### CODE REVIEW

**Comment by eri9600** [comment]
Hi @TADraeseke,
Please review when you have time. Thanks!

File: java/spatial-relationships/src/main/java/com/esri/arcgisruntime/sample/spatialrelationships/MainActivity.java [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+            (int) motionEvent.getY());
+        ListenableFuture<IdentifyGraphicsOverlayResult> identifyGraphics =
+            mMapView.identifyGraphicsOverlayAsync(graphicsOverlay, clickLocation, 1, false);
+        identifyGraphics.addDoneListener(() -> {
+
+          try {
+            // get the first identified graphic
+            IdentifyGraphicsOverlayResult result = identifyGraphics.get();
+            List<Graphic> identifiedGraphics = result.getGraphics();
+            if (identifiedGraphics.size() > 0) {
```
Comment (TADraeseke) [inline java/spatial-relationships/src/main/java/com/esri/arcgisruntime/sample/spatialrelationships/MainActivity.java]: can replace with if (!identifiedGraphics.isEmpty()) {

File: java/spatial-relationships/src/main/java/com/esri/arcgisruntime/sample/spatialrelationships/MainActivity.java [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+    });
+  }
+
+  /**
+   * Gets a string representation of the spatial relationship list
+   *
+   * @param relationshipList a list of spatial relationships
+   * @return a string list of spatial relationships
+   */
+  private ArrayList<String> relationshipStringList(List<QueryParameters.SpatialRelationship> relationshipList) {
```
Comment (TADraeseke) [inline java/spatial-relationships/src/main/java/com/esri/arcgisruntime/sample/spatialrelationships/MainActivity.java]: method can be static

File: java/spatial-relationships/src/main/java/com/esri/arcgisruntime/sample/spatialrelationships/MainActivity.java [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+  }
+
+  /**
+   * Gets a list of spatial relationships that the first geometry has to the second geometry.
+   *
+   * @param a first geometry
+   * @param b second geometry
+   * @return list of relationships a has to b
+   */
+  private List<QueryParameters.SpatialRelationship> getSpatialRelationships(Geometry a, Geometry b) {
```
Comment (TADraeseke) [inline java/spatial-relationships/src/main/java/com/esri/arcgisruntime/sample/spatialrelationships/MainActivity.java]: can be static

File: java/spatial-relationships/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+<resources>
+    <string name="app_name">Spatial Relationships</string>
+    <string name="Relationships"> Relationships</string>
```
Comment (TADraeseke) [inline java/spatial-relationships/src/main/res/values/strings.xml]: Is the space needed before Relationships?

**Comment by eri9600** [comment]
@TADraeseke I fixed the changes you suggested. Merging now.

**Pull request merged**